transportable-udfs-test/transportable-udfs-test-spi/src/main/java/com/linkedin/transport/test/spi/SqlFunctionCallGenerator.java

```diff
@@ -85,6 +88,10 @@ default String getStringArgumentString(String value) {
     return "'" + value + "'";
   }
 
+  default String getDoubleArgumentString(Double value) {
+    return String.valueOf(value);
```

Comment:
Do we need  similar methods for the other two new types?

Reply:
Same answer as above. SQL text seems not support Binary and Float literals.